stdlib/3/datetime.pyi

```diff
@@ -197,7 +197,7 @@ class datetime:
     def timetz(self) -> _time: ...
     def replace(self, year: int = ..., month: int = ..., day: int = ..., hour: int = ...,
                 minute: int = ..., second: int = ..., microsecond: int = ..., tzinfo:
-                Union[_tzinfo, bool] = ...) -> datetime: ...
+                Union[_tzinfo, bool, None] = ...) -> datetime: ...
```

Comment:
Interesting -- do you understand why `bool` is allowed here already? I don't see it in the doc you linked to.

@matthiaskramm, it looks like you wrote this line in dd042b325. Do you recall why you included `bool` here?